lib/_tls_wrap.js

```diff
@@ -199,6 +199,20 @@ function onocspresponse(resp) {
   this.emit('OCSPResponse', resp);
 }
 
+function initread(tls, wrapped) {
```

Comment:
Gotcha. I was trying to follow the function above `onnewsession` which was all lowercase so I thought that was the standard :/